lib/routes/routes.dart

```diff
@@ -9,7 +10,8 @@ import 'package:flutter/material.dart';
 import 'package:go_router/go_router.dart';
 
 import '../views/home/home_wrapper_widget.dart';
-import '../views/player/player2/player_view.dart';
+import '../views/player_old/player2/player_view.dart';
+// import '../views/player/player_view.dart';
```

Comment:
Remove comment?

Reply:
Actually, I was switching between the screens that's why I didn't remove the comment but we don't need it anymore so, I will remove all the comments thanks.